rotkehlchen/db/schema.py

```diff
@@ -825,7 +825,18 @@
     active INTEGER NOT NULL CHECK (active IN (0, 1)),
     weight INTEGER NOT NULL
 );
-"""  # noqa: E501
+"""
+
+DB_CREATE_USER_NOTES = """
+CREATE TABLE IF NOT EXISTS user_notes(
+    identifier INTEGER NOT NULL PRIMARY KEY,
+    title TEXT NOT NULL,
+    content TEXT NOT NULL,
+    location TEXT NOT NULL,
+    last_update_timestamp INTEGER NOT NULL,
+    is_pinned INTEGER NOT NULL CHECK (is_pinned IN (0, 1))
```

Comment:
What is this???

Reply:
This is a field requested by luki to mark the notes as pinned so he can show them in the UI